products/db-knownprojects/sca_aggregate/create_zap_projects.sql

```diff
@@ -1,35 +1,40 @@
 --- Get dcp_bblnumber and dcp_name from dcp_projectbbls
 DROP TABLE IF EXISTS dcp_projectbbl_subset;
 CREATE TABLE dcp_projectbbl_subset AS
-SELECT 
-     -- Extract the first 9 characters of the "dcp_name" column and rename it as "dcp_name" to match 
-     SUBSTRING(dcp_name FROM 1 FOR 9) AS dcp_name,
-     -- Rename dcp_bblnumber to bbl 
-     dcp_bblnumber AS bbl
+SELECT
+    -- Extract the first 9 characters of the "dcp_name" column and rename it as "dcp_name" to match 
+    dcp_bblnumber AS bbl,
```

Comment:
Did fluff reorder this query or was that you?

Reply:
pretty sure this was fluff. I'll look for the "rule" but I think a convention is to put function's last so 

```
SUBSTRING(dcp_name FROM 1 FOR 9) AS dcp_name,
dcp_bblnumber AS bbl
```

becomes

```
dcp_bblnumber AS bbl,
SUBSTRING(dcp_name FROM 1 FOR 9) AS dcp_name
```

Reply:
found the rule! it's [`structure.column_order`](https://docs.sqlfluff.com/en/stable/rules.html#rule-structure.column_order)

"Select wildcards then simple targets before calculations and aggregates"